EpsilonLights/Src/Lights.c

```diff
@@ -154,6 +151,44 @@ void blinkSignalLightsTask(void const* arg)
     }
 }
 
+void updateStrobeLight(void const* arg)
+{
+    uint32_t prevWakeTime = osKernelSysTick();
+    // Store inputs values
+    // NOTE: All Lights Out pins are active low
+
+    // If blinkerTimer is within (0 - BLINKER_FREQ), turn blinkers on
+    // If blinkerTimer is within (BLINKER_FREQ - BLINKER_FREQ*2), keep blinkers off
+    // If blinkerTimer is greater than (BLINKER_FREQ*2) reset blinkerTimer to 0
+    uint32_t blinkerTimer = 0;
+
+    for (;;)
+    {
+        osDelayUntil(&prevWakeTime, LIGHTS_UPDATE_FREQ);
+
+        /*Update BMS Strobe*/
+        if (auxBmsInputs[1] & STROBE_FAULT_MASK && blinkerTimer >= BLINKER_FREQ)
+        {
+            HAL_GPIO_WritePin(ESTROBE_GPIO_Port, ESTROBE_Pin, LIGHT_ON);
+        }
+        else
+        {
+            HAL_GPIO_WritePin(ESTROBE_GPIO_Port, ESTROBE_Pin, LIGHT_OFF);
+        }
+
+        // Update blinker timer
+        if (blinkerTimer > BLINKER_FREQ * 2)
```

Comment:
This doesn't need to be continuously updated, having the timer run continuously also prevents the light from coming on right away. If a fault stage is hit, we should see the strobe light up right away, not the next cycle of the blinker timer.